### PR TITLE
Target Rider 2023.1

### PR DESCRIPTION
**Pull request by rafaelldi**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties [hunk 1]
```diff
@@ -5,14 +5,14 @@ org.gradle.jvmargs='-Duser.language=en'
 sources=false
 
 intellij_version=IC-231.8109-EAP-CANDIDATE-SNAPSHOT
```
Comment (rafaelldi) [inline PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties:7]: Where can I find the right version number?

Comment (maartenba) [reply PluginsAndFeatures/azure-toolkit-for-intellij/gradle.properties:7]: I usually pick the latest snapshot from https://www.jetbrains.com/intellij-repository/snapshots that has a version number as close to Rider's as possible.

**Comment by rafaelldi** [comment]
I have some compilation problems =( 
I am going to investigate how to solve them.

**Comment by maartenba** [comment]
Let me know if you need another pair of 👀

**Review by maartenba — APPROVED** [review]

**Pull request merged**